src/timeMachine/actions/queries.ts

```diff
@@ -319,13 +295,12 @@ export const executeQueries =
 
       await dispatch(hydrateVariables())
       const allVariables = getAllVariables(state)
-      const allBuckets = getAll<Bucket>(state, ResourceType.Buckets)
 
       const startTime = window.performance.now()
       const startDate = Date.now()
 
       const pendingResults = queries.map(({text}) => {
-        const orgID = getOrgIDFromBuckets(text, allBuckets) || getOrg(state).id
+        const orgID = getOrg(state).id
```

Comment:
Same story here about calling `getOrg(state).id` outside of the queries loop. See [this line.](https://github.com/influxdata/ui/commit/45d1b2d30a5f0481a5db56bb9a31e3d07cbf2e3b#diff-0f78a1f05484420b38348fd715c2179c9f471f3ae36ba25dec94d60f5529d1edL139)